jenkins/cut-release

```diff
@@ -81,11 +86,13 @@ update_and_or_cut_branch_for_repository() {
         log_new_commits "${repo}"
         if ${SHOULD_CUT_BRANCH}; then
             cut_branch "${repo}" "${BRANCH_OLD}" "${BRANCH_NEW}" "${MERGE_MESSAGE}"
-            ${WORKSPACE}/send-email.py \
-                "${EMAIL_RECIPIENT}" \
-                "${EMAIL_SUBJECT}" \
-                "${output_file}" \
-            ;
+            if [ -n "${EMAIL_RECIPIENT}" ]; then
```

Comment:
Only send an email if an address has been specified.
By default, when running locally, no email is sent.